src/main.rs

```diff
@@ -0,0 +1,67 @@
+extern crate serde_json;
+extern crate serde;
+
+
+const ACCOUNTS: i32 = 8;
```

Comment:
Would it be possible to import these constants from the same file?

Reply:
Yeah will do it.